python/README.md

```diff
@@ -0,0 +1,59 @@
+# SQLFlow Playground Server
```

Comment:
It's an excellent idea, maybe we should push a design doc to introduce the Playground server and what APIs the server exposed, and then push  the implementation PRs.

please @typhoonzero also takes a  look at  the PR.

Reply:
Why not use sqlflowserver grpc service directly?

Reply:
Is this used for aliyun DSW only?

Reply:
Current it's only used by DSW. We need to create a DB pod for each user, and kill the pod after some idle time.

Reply:
done